Automattic/wp-calypso
# Pattern Library: Use real pattern previews in category links

Fixes https://github.com/Automattic/dotcom-forge/issues/5869

## Proposed Changes

On the `/patterns` landing page, there is a grid with category links. Each link contains a pattern preview. Up until now, we've used screenshots for these previews. This PR makes it so that we now use dynamically rendered pattern previews instead.

**🚨 NOTE:** the category preview patterns are not yet manually selected. I realize this is something we probably want to implement, but for now, we just pick the first pattern from every category.

<img src="https://github.com/Automattic/wp-calypso/assets/1101677/a4bd655e-ee34-4ea3-80f7-9d6835e74eaa" width="350" />

## Testing Instructions



1. Apply D141467-code
2. Sandbox `public-api.wordpress.com`
3. Navigate to `/patterns` 
4. Ensure that every category shows a different preview pattern
5. Ensure that the page matches the Figma mockups BnTfb6hzKcRTb2LBtuPwKe-fi-246%3A42066

## Pre-merge Checklist
